lib/Sips/ShaComposer/OfficeShaComposer.php

```diff
@@ -1,41 +1,35 @@
 <?php
+declare(strict_types=1);
 
 namespace Sips\ShaComposer;
 
 use Sips\Passphrase;
 
-class OfficeShaComposer implements ShaComposer
+final class OfficeShaComposer implements ShaComposer
 {
-    /**
-     * @var string Passphrase
-     */
-    private $passphrase;
+    private string $passphrase;
+    private array $ignoredParameters = ['keyVersion', 'seal'];
 
-    private $ignoredParameters = array('keyVersion', 'seal');
-
-    /**
-     * @param \Sips\Passphrase $passphrase
-     */
     public function __construct(Passphrase $passphrase)
     {
-        $this->passphrase = $passphrase;
+        $this->passphrase = $passphrase->__toString();
```

Comment:
Shouldn't it be better to create an additional method without the underscores if you want to use it explicitly?
```suggestion
        $this->passphrase = $passphrase->toString();
```